File: app/src/main/java/knf/kuma/tv/cards/RecentsCardView.java
package knf.kuma.tv.cards;

import android.content.Context;
import android.widget.ImageView;
import android.widget.TextView;

import butterknife.BindView;
import butterknife.ButterKnife;
import knf.kuma.R;
import knf.kuma.commons.PicassoSingle;
import knf.kuma.pojos.RecentObject;
import knf.kuma.tv.BindableCardView;

public class RecentsCardView extends BindableCardView<RecentObject> {

    @BindView(R.id.img)
    ImageView imageView;
    @BindView(R.id.title)
    TextView title;
    @BindView(R.id.chapter)
    TextView chapter;

    public RecentsCardView(Context context) {
        super(context);
        ButterKnife.bind(this);
    }

    @Override
    public void bind(RecentObject data) {
        PicassoSingle.get(context).load(data.img).into(imageView);
        title.setText(data.name);
        chapter.setText(data.chapter);
    }

    @Override
    public ImageView getImageView() {
        return imageView;
    }

    @Override
    public int getLayoutResource() {
        return R.layout.item_tv_card_chapter;
    }
}

File: app/src/main/java/knf/kuma/tv/cards/RecentsCardView.kt
package knf.kuma.tv.cards

import android.content.Context
import android.widget.ImageView
import android.widget.TextView
import knf.kuma.R
import knf.kuma.commons.loadGlide
import knf.kuma.pojos.av1.RecentAV1
import knf.kuma.tv.BindableCardView
import org.jetbrains.anko.find

class RecentsCardView(context: Context) : BindableCardView<RecentAV1>(context) {

    override val imageView: ImageView
        get() = find(R.id.img)
    override val layoutResource: Int
        get() = R.layout.item_tv_card_chapter

    override fun bind(data: RecentAV1) {
        imageView.loadGlide(data.animeImageUrl)
        find<TextView>(R.id.title).text = data.name
        find<TextView>(R.id.chapter).text = data.chapter
    }
}
